<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgPanelUse</class>
 <widget class="QDialog" name="dlgPanelUse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel usage</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="gbPanelData">
       <property name="title">
        <string> Panel data </string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblPanelName">
          <property name="text">
           <string>Name :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblPanelTitle">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbPatientCard">
       <property name="title">
        <string> Use with patientcard ... </string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblPatientCardBarcode">
            <property name="text">
             <string>Barcode :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ledPatientCardBarcode"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbPatientCardUnits">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="title">
        <string> Available units </string>
       </property>
       <layout class="QVBoxLayout" name="vlUnits">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="vlUnits_2">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="gbPanelUseSummary">
       <property name="title">
        <string> Usage summary </string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lblTotalTime">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt;</string>
          </property>
          <property name="text">
           <string>Total time :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblTotalTimeValue">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt;
color: rgb(0, 125, 0);</string>
          </property>
          <property name="text">
           <string>00 : 00 : 00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblTotalPrice">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt;</string>
          </property>
          <property name="text">
           <string>Total price :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblTotalPriceValue">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt;
color: rgb(0, 125, 0);</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>162</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pbOk</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>ledPatientCardBarcode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
